source/mod_hash.f90

```diff
@@ -116,10 +110,8 @@ subroutine hash_initialise
     tStatus = "Failed"
     hash_selfTestOK = .false.
   end if
-  if(st_debug) then
-    write(lout,"(a)") "HASH>   #3 : "//md5Digest//" < 'abcdefghijklmnopqrstuvwxyz'"
-    write(lout,"(a)") "HASH>    3 : "//md5Valid//" > "//tStatus
-  end if
+  ! write(lout,"(a)") "HASH>   #3 : "//md5Digest//" < 'abcdefghijklmnopqrstuvwxyz'"
+  ! write(lout,"(a)") "HASH>    3 : "//md5Valid//" > "//tStatus
```

Comment:
They're development debugging info. The result of the fail test is reported in the final file, so it is not necessary for runtime.

I decided also to remove it from the global debug flag if statement because this is printed during initialisation, which causes a few issues as this is called very early in maincr.